examples/simulated/plot_toy_classification.py

```diff
@@ -0,0 +1,73 @@
+"""
+=====================================================================
+Illustrate classification accuracy versus class separability
+=====================================================================
+
+Generate several datasets containing data points from two-classes. Each class
+is generated with a Riemannian Gaussian distribution centered at the class mean
+and with the same dispersion sigma. The distance between the class means is
+parametrized by Delta, which we make vary between zero and 5*sigma. We
+illustrate how the accuracy of the MDM classifier varies when Delta increases.
+
+"""
+# Authors: Pedro Rodrigues <pedro.rodrigues@melix.org>
+#
+# License: BSD (3-clause)
+
+import numpy as np
+import matplotlib.pyplot as plt
+from sklearn.model_selection import cross_val_score
+
+from pyriemann.classification import MDM
+from pyriemann.datasets import make_gaussian_blobs
+
+
+print(__doc__)
+
+
+###############################################################################
+# Set general parameters for the illustrations
+
+
+n_dim = 4  # dimensionality of the data points
+sigma = 1.0  # dispersion of the Gaussian distributions
+random_state = 42  # ensure reproducibility
+
+###############################################################################
+# Loop over different levels of separability between the classes
+scores_array = []
+deltas_array = np.linspace(0, 5*sigma, 5)
+
+for delta in deltas_array:
+    # generate data points for a classification problem
+    X, y = make_gaussian_blobs(n_matrices=100,
+                               n_dim=n_dim,
+                               class_sep=delta,
+                               class_disp=sigma,
+                               random_state=random_state)
+
+    # which classifier to consider
+    clf = MDM()
+
+    # get the classification score for this setup
+    scores_array.append(
+        cross_val_score(clf, X, y, cv=5, scoring='roc_auc').mean())
+
+scores_array = np.array(scores_array)
+
+###############################################################################
+# Plot the results
+fig, ax = plt.subplots(figsize=(7.5, 5.9))
+ax.plot(deltas_array, scores_array, lw=3.0, label=sigma)
```

Comment:
Is it possible to display a figure with several values of sigma (for eg, `sigma=0.5`, `sigma=1` and `sigma=2`)?

Reply:
The x-axis is epsilon/sigma, so the curves would all be the same except for the intrinsic variability of the sampling procedure. Moreover, we would go from ~15 s to ~50 s of execution time. I am not sure if it is really worth it.

Reply:
Ok, you are right.